sos/plugins/kernel.py

```diff
@@ -92,4 +95,11 @@ def setup(self):
             clocksource_path + "current_clocksource"
         ])
 
+        if self.get_option("with-timer"):
+            # This can be very slow, depending on the number of timers,
+            # and may also cause softlockups
+            self.add_copy_spec([
+                "/proc/timer*"
+            ])
```

Comment:
Please don't use a list for a single item or break the call over multiple lines: this can simply be written:

```python
        self.add_copy_spec("/proc/timer*")
```